modules/assim.sequential/NAMESPACE

```diff
@@ -61,7 +61,6 @@ export(tobit_model_censored)
 export(y_star_create)
 import(furrr)
 import(lubridate)
-import(ncdf4)
```

Comment:
I think this showing up here is an artifact of when you last merged develop, and will probably disappear when you merge again. No action needed unless it's still here after that.